### PR TITLE
Strategy for Jelly veNFTs [jelly-ve-nft-balance]

### PR DESCRIPTION
**Pull request by ProfWobble**
Changes proposed in this pull request:

Add support for Jelly veNFT based voting strategy

### CODE REVIEW

File: src/strategies/jelly-ve-nft-balance/index.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'JellyProtocol';
+export const version = '0.1.3';
```
Comment (ChaituVR) [inline src/strategies/jelly-ve-nft-balance/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hey @ProfWobble you can use `contract-call` strategy for this

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Like this 
https://snapshot.org/#/playground/contract-call?query=eyJwYXJhbXMiOnsiYWRkcmVzcyI6IjB4MmUxZTg1YjVjMGYzMzc1ZjI2NGEzMTQ4MThlMjU1Y2VlYzZiMWE2MCIsInN5bWJvbCI6IkNha2UiLCJkZWNpbWFscyI6MTgsIm1ldGhvZEFCSSI6eyJpbnB1dHMiOlt7ImludGVybmFsVHlwZSI6ImFkZHJlc3MiLCJuYW1lIjoiYWNjb3VudCIsInR5cGUiOiJhZGRyZXNzIn1dLCJuYW1lIjoidm90ZXJCYWxhbmNlIiwib3V0cHV0cyI6W3siaW50ZXJuYWxUeXBlIjoidWludDI1NiIsIm5hbWUiOiIiLCJ0eXBlIjoidWludDI1NiJ9XSwic3RhdGVNdXRhYmlsaXR5IjoidmlldyIsInR5cGUiOiJmdW5jdGlvbiJ9fSwibmV0d29yayI6IjQiLCJzbmFwc2hvdCI6MTExMjU2OTAsImFkZHJlc3NlcyI6WyIweDdlZkRFYTU2MDg4RTFkZTRjYTEyOGQxQUNmZURhYkIwNjcyMDJlNTYiXX0.

File: src/strategies/jelly-ve-nft-balance/index.ts (hunk 1 above)
Comment (ProfWobble) [reply src/strategies/jelly-ve-nft-balance/index.ts:6]: I'm going to make this strategy more complicated once I finish implementing our veNFTs. I wanted to have this strategy so I can test

**Comment by ProfWobble** [comment]
I understand, however I'm building a permissionless contract factory, with Snapshot out of the box. I only want users to input their veNFT contract via schema.json when they add the Jelly strategy. When I finish the veNFT contract, it will pull more defaults from the both the ve contracts and connected airdrops. I dont want new users to have to configure custom contract-call strategies.

**Review by ProfWobble — COMMENTED** [review]
reviewed

**Comment by ChaituVR** [comment]
I can understand but the main idea is to avoid duplication, when will you add more settings to this strategy? better to merge it after that, once spaces start using this strategy. it is kind of complicated to update strategy, specially when there is a open proposal

**Comment by ProfWobble** [comment]
Good to know. We plan to do a public testnet with our community as soon as this is merged. With any changes being made after community feedback and before we launch on mainnet.

**Comment by ChaituVR** [comment]
Closing this for now, i expect that contract-call worked for you like mention above, please feel free to reopen if needed

**Pull request closed without merging**